--- /etc/logrotate.conf.orig	2007-08-29 02:19:36.000000000 -0500
+++ /etc/logrotate.conf	2012-02-04 15:05:45.223379489 -0600
@@ -12,7 +12,10 @@
 dateext
 
 # uncomment this if you want your log files compressed
-#compress
+compress
+
+# rotate old log files to one directory
+olddir /var/log/rotated
 
 # RPM packages drop log rotation information into this directory
 include /etc/logrotate.d
@@ -21,7 +24,6 @@
 /var/log/wtmp {
     monthly
     create 0664 root utmp
-	minsize 1M
     rotate 1
 }
 
