--- /etc/logrotate.d/syslog.org	2012-02-04 14:56:54.638381783 -0600
+++ /etc/logrotate.d/syslog	2012-02-04 14:20:49.527376475 -0600
@@ -1,8 +1,4 @@
-/var/log/cron
-/var/log/maillog
-/var/log/messages
-/var/log/secure
-/var/log/spooler
+/var/log/auth.log /var/log/authpriv.log /var/log/cron.log /var/log/daemon.log /var/log/kern.log /var/log/lpr.log /var/log/mail.log /var/log/news.log /var/log/syslog.log /var/log/user.log /var/log/uucp.log /var/log/local.log /var/log/boot.log 
 {
     sharedscripts
     postrotate
