--- template64/etc/profile	2009-07-02 15:41:15.000000000 -0500
+++ template32/etc/profile	2009-09-04 06:13:12.000000000 -0500
@@ -19,12 +19,9 @@
 	UID=`id -ru`
 fi
 
-# Path manipulation
-if [ "$EUID" = "0" ]; then
-	pathmunge /sbin
-	pathmunge /usr/sbin
-	pathmunge /usr/local/sbin
-fi
+pathmunge /sbin
+pathmunge /usr/sbin
+pathmunge /usr/local/sbin
 
 # No core files by default
 ulimit -S -c 0 > /dev/null 2>&1
