--- template64/etc/sudoers	2009-07-02 15:54:15.000000000 -0500
+++ template32/etc/sudoers	2009-08-04 09:31:04.000000000 -0500
@@ -80,7 +80,7 @@
 # %sys ALL = NETWORKING, SOFTWARE, SERVICES, STORAGE, DELEGATING, PROCESSES, LOCATE, DRIVERS
 
 ## Allows people in group wheel to run all commands
-# %wheel	ALL=(ALL)	ALL
+%wheel	ALL=(ALL)	ALL
 
 ## Same thing without a password
 # %wheel	ALL=(ALL)	NOPASSWD: ALL

